<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MVStudioClass</class>
 <widget class="QMainWindow" name="MVStudioClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MVStudio</string>
  </property>
  <property name="iconSize">
   <size>
    <width>28</width>
    <height>28</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="toolBarFile">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>File</string>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProject"/>
   <addaction name="actionOpenProject"/>
   <addaction name="actionSaveProject"/>
   <addaction name="separator"/>
   <addaction name="actionSnapshot"/>
  </widget>
  <widget class="QToolBar" name="toolBarView">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>View</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFullScreen"/>
   <addaction name="actionFitWindow"/>
   <addaction name="actionPerspectiveOrthographic"/>
   <addaction name="actionShowCoordinateSystem"/>
   <addaction name="actionLighting"/>
   <addaction name="separator"/>
   <addaction name="actionPointAsSphere"/>
   <addaction name="actionIncreasePointSize"/>
   <addaction name="actionDecreasePointSize"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTest"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>38</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionSaveProjectAs"/>
    <addaction name="separator"/>
    <addaction name="actionOpenPointCloud"/>
    <addaction name="actionSavePointCloud"/>
    <addaction name="actionCombinePointCloudFiles"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFullScreen"/>
    <addaction name="actionFitWindow"/>
    <addaction name="separator"/>
    <addaction name="actionPerspectiveOrthographic"/>
    <addaction name="actionShowCoordinateSystem"/>
    <addaction name="actionLighting"/>
    <addaction name="separator"/>
    <addaction name="actionIncreasePointSize"/>
    <addaction name="actionDecreasePointSize"/>
    <addaction name="actionPointAsSphere"/>
    <addaction name="separator"/>
    <addaction name="actionSetBackgroundColor"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionSnapshot"/>
    <addaction name="separator"/>
    <addaction name="actionCopyCamera"/>
    <addaction name="actionPasteCamera"/>
    <addaction name="separator"/>
    <addaction name="actionSaveCameraToFile"/>
    <addaction name="actionRestoreCameraFromFile"/>
   </widget>
   <widget class="QMenu" name="menuReconstruction">
    <property name="title">
     <string>Reconstruct</string>
    </property>
    <addaction name="actionImageMatching"/>
    <addaction name="separator"/>
    <addaction name="actionSparseReconstruction"/>
    <addaction name="actionDenseReconstruction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuCamera"/>
   <addaction name="menuReconstruction"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetImages">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>224</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Images</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetModelContents">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="WidgetImageList" name="listWidgetImages">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
       <property name="selectionRectVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetOutput">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>182</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEditOutput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/exit.png</normaloff>:/Resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/fullscreen.png</normaloff>:/Resources/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="toolTip">
    <string>Show full screen</string>
   </property>
   <property name="statusTip">
    <string>Show full screen</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionFitWindow">
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/home.png</normaloff>:/Resources/home.png</iconset>
   </property>
   <property name="text">
    <string>Fit Window</string>
   </property>
   <property name="toolTip">
    <string>Show entire scene</string>
   </property>
   <property name="statusTip">
    <string>Show entire scene</string>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/snap_shot.png</normaloff>:/Resources/snap_shot.png</iconset>
   </property>
   <property name="text">
    <string>Snapshot</string>
   </property>
  </action>
  <action name="actionPerspectiveOrthographic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/perspective.png</normaloff>:/Resources/perspective.png</iconset>
   </property>
   <property name="text">
    <string>Perspective / Orthographic</string>
   </property>
  </action>
  <action name="actionSaveCameraToFile">
   <property name="text">
    <string>Save Camera to File</string>
   </property>
   <property name="iconText">
    <string>Save Camera State</string>
   </property>
   <property name="toolTip">
    <string>Save Camera State</string>
   </property>
  </action>
  <action name="actionRestoreCameraFromFile">
   <property name="text">
    <string>Restore Camera from File</string>
   </property>
   <property name="iconText">
    <string>Load Camera State</string>
   </property>
  </action>
  <action name="actionShowCoordinateSystem">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/axis_and_label.png</normaloff>:/Resources/axis_and_label.png</iconset>
   </property>
   <property name="text">
    <string>Show Coordinate System</string>
   </property>
  </action>
  <action name="actionOpenPointCloud">
   <property name="text">
    <string>Open Point Cloud</string>
   </property>
  </action>
  <action name="actionSavePointCloud">
   <property name="text">
    <string>Save Point Cloud</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionSetBackgroundColor">
   <property name="text">
    <string>Set Background Color</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/help.png</normaloff>:/Resources/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionCopyCamera">
   <property name="text">
    <string>Copy Camera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPasteCamera">
   <property name="text">
    <string>Paste Camera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSparseReconstruction">
   <property name="text">
    <string>Run Sparse (SfM)</string>
   </property>
  </action>
  <action name="actionDenseReconstruction">
   <property name="text">
    <string>Run Dense (MVS)</string>
   </property>
  </action>
  <action name="actionCombinePointCloudFiles">
   <property name="text">
    <string>Combine Point Clouds</string>
   </property>
  </action>
  <action name="actionImageMatching">
   <property name="text">
    <string>Image Matching</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/fileopen.png</normaloff>:/Resources/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/filesave.png</normaloff>:/Resources/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/new.png</normaloff>:/Resources/new.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPointAsSphere">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/spheres.png</normaloff>:/Resources/spheres.png</iconset>
   </property>
   <property name="text">
    <string>Point As Sphere</string>
   </property>
  </action>
  <action name="actionIncreasePointSize">
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/plus.png</normaloff>:/Resources/plus.png</iconset>
   </property>
   <property name="text">
    <string>Increase Point Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionDecreasePointSize">
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/minus.png</normaloff>:/Resources/minus.png</iconset>
   </property>
   <property name="text">
    <string>Decrease Point Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionLighting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources/MVStudio.qrc">
     <normaloff>:/Resources/lighting.png</normaloff>:/Resources/lighting.png</iconset>
   </property>
   <property name="text">
    <string>Lighting</string>
   </property>
  </action>
  <action name="actionSaveProjectAs">
   <property name="text">
    <string>Save Project As</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WidgetImageList</class>
   <extends>QListWidget</extends>
   <header>widget_image_list.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dockWidgetImages</tabstop>
  <tabstop>listWidgetImages</tabstop>
  <tabstop>dockWidgetOutput</tabstop>
  <tabstop>plainTextEditOutput</tabstop>
 </tabstops>
 <resources>
  <include location="Resources/MVStudio.qrc"/>
 </resources>
 <connections/>
</ui>
